cortex/kernel_features/llm_device.py

```
@@ -0,0 +1,142 @@
+        if t == 'session' and not m: return base + list(self.sessions.keys())
+        if t == 'session' and m: return base + ['prompt', 'response', 'history']
+        return base
+    
+    def read(self, path, size, offset, fh):
+        t, m, f = self._parse(path)
+        s = self.sessions.get("default")
+        if t == 'model' and f == 'response': return s.response.encode()[offset:offset+size]
+        if t == 'status': return json.dumps({"status": "running", "uptime": time.time() - self.start, "requests": self.requests}).encode()[offset:offset+size]
+        return b""
+    
+    def write(self, path, data, offset, fh):
```

Ruff (0.14.7)

Unused method argument: `offset`

(ARG002)